static/app/views/issueDetails/groupSimilarIssues/similarStackTrace/toolbar.tsx

```diff
@@ -63,6 +67,9 @@ class SimilarToolbar extends Component<Props, State> {
           parentGroupId,
           groupId: this.props.groupId,
           value,
+          wouldGroup: itemWouldGroup?.length
+            ? itemWouldGroup[0].shouldBeGrouped
+            : undefined,
```

Comment:
nit should be the same
```suggestion
          wouldGroup: itemWouldGroup[0]?.shouldBeGrouped,
```